src/javascript/app/pages/cashier/cashier.js

```diff
@@ -1,10 +1,14 @@
+/* eslint-disable no-nested-ternary */
 const Client           = require('../../base/client');
 const BinarySocket     = require('../../base/socket');
 const isCryptocurrency = require('../../common/currency').isCryptocurrency;
+const elementInnerHtml = require('../../../_common/common_functions').elementInnerHtml;
 const getElementById   = require('../../../_common/common_functions').getElementById;
 const localize         = require('../../../_common/localize').localize;
+const State            = require('../../../_common/storage').State;
 const paramsHash       = require('../../../_common/url').paramsHash;
 const urlFor           = require('../../../_common/url').urlFor;
+const urlForStatic     = require('../../../_common/url').urlForStatic;
```

Comment:
may be worth importing the whole url file and calling the functions since we are using three of them?